src/app/state/network/saga.test.ts

```diff
@@ -0,0 +1,15 @@
+import { testSaga } from 'redux-saga-test-plan'
+import { networkActions } from '.'
+import { networkSaga, selectNetwork } from './saga'
+
+describe('Network Sagas', () => {
+  test('networkSaga', () => {
+    testSaga(networkSaga)
+      .next()
+      .takeLatest(networkActions.selectNetwork, selectNetwork)
+      .next()
+      .put(networkActions.selectNetwork(process.env.REACT_APP_LOCALNET ? 'local' : 'mainnet'))
```

Comment:
@lukaw3d  Do we have some pattern for handling such cases? ('local' for CLI, 'mainnet' for dev)
Other options are: override process env in setupTests or locally in tests, right?